### ISSUE TITLE
docker bind mount directory not found 

### ISSUE THREAD

**Issue by aakhter** [FallingSnow/h265ize#139]
I'm getting a directory not found on h265ize invocation. This used to work fine, but I had to rebuild part of the setup and something seems to have gone wrong... Appreciate any thoughts.

* The directory exists, and in fact invoking h265ize with a specific file in that directory works fine. 
* **The directory is a docker bind mount. Other directories seem to work fine**

root@86ad6249a999:/h265ize# **h265ize /input -d /output**
(node:2039) UnhandledPromiseRejectionWarning: **Error: File or directory /input does not exist.**
    at /usr/local/lib/node_modules/h265ize/lib/helpers.js:198:35
    at /usr/local/lib/node_modules/h265ize/node_modules/recursive-readdir/index.js:75:22

**Comment by FallingSnow** [comment]
Try it on a non bind directory. Does it still error out?

**Comment by aakhter** [comment]
I played around with it a bit the problem went away. I'm not sure what was causing it. At the time I had tried a different (non-bind) dir and had not seen the problem. However, the contents of that dir were different. 

So it could have been:
* something weird in the directory (the contents have since changed)
* something about how the container was built (I have since rebuilt it)

That said, I had been meaning to add a blacklist capability and finer control over the queue- so wound up building a quick harness around h265ize that only feeds a single file at a time. 
https://bitbucket.org/aamer_akhter/h265ize-qnap-nvidia/src/master/h265-watch.py
